solutions/symmetric_substrings_solution.rb

```diff
@@ -1,4 +1,10 @@
 class String
+  # Define a method `symmetric_substrings` that returns 
```

Comment:
Should be `String#symmetric_substrings` or otherwise specify that it's monkey patched onto the String class